tech.json

```diff
@@ -320,7 +320,10 @@
                   "extensionTechs": [
                     {
                       "name": "canInsaneWalljump",
-                      "requires": [ "canDelayedWalljump" ],
+                      "requires": [
+                        "canDelayedWalljump",
+                        "canConsecutiveWalljump"
```

Comment:
`canInsaneWallJump` is better left describing the jump itself.  Many cases are 2 jumps total, taco tank is 1 walljump.

Reply:
I was planning on changing this to canStaggeredWalljump once that tech is merged in. I figure it would take a couple simple wall  jumps to position Samus. Taco Tank and Screw CWJ  don't need that though ....